crates/hyperswitch_domain_models/src/payments.rs

```diff
@@ -80,6 +80,51 @@ impl PaymentIntent {
     pub fn get_id(&self) -> &id_type::PaymentId {
         &self.payment_id
     }
+    #[cfg(feature = "v1")]
+    pub fn get_order_tax_amount(&self, payment_attempt: &PaymentAttempt) -> Option<MinorUnit> {
```

Comment:
can you add this function on the OperationSessionGetters trait? Since you need both intent and attempt, both should be available in payment data